app/views/wishlists/index.html.erb

```diff
@@ -3,12 +3,13 @@
   <ul>
     <% @wishlists.each do |wishlist| %>
       <li>
-        <%= link_to wishlist.name, "#" -%>
-        <%= button_to "Delete", wishlist, method: :delete -%>
+        <%= link_to wishlist.name, "#" %>
+        <%= button_to "Edit", edit_wishlist_path(wishlist), method: :get %>
```

Comment:
`link_to` is enough. With `button_to` you're sending a POST request from a form, that later gets changed to a GET.